### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by mifrazmurthaja**
Add docs for JWT SSO

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## JWT SSO Inbound Authenticator
+The JWT SSO Inbound Authenticator is an inbound authentication protocol to implement JWT based SSO for your application 
+with WSO2 Identity Server.
+
+## Getting started
+To get started with the JWT SSO Inbound Authenticator, go to Configuring [JWT SSO Inbound Authenticator](config.md).
+Once you have completed your configurations, you can perform single-sign-on for the web applications which uses
```
Comment (gomathyk) [inline docs/README.md]: Shall we replace this as follows? 

web applications that use the JWT-based SSO protocol.

File: docs/config.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# Configuring JWT SSO Inbound Authenticator
+
+This topic provides instructions on configuring the JWT SSO inbound authenticator and the WSO2 Identity Server 
+to perform single-sign-on with the applications which use JWT based SSO.
```
Comment (gomathyk) [inline docs/config.md]: JWT-based

File: docs/config.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+### Download and set up JWT SSO
+
+* [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management) and
+[Install the Product](https://is.docs.wso2.com/en/5.10.0/setup/installing-the-product).
+* Download the JWT SSO Inbound Authenticator JAR from the 
+[IS Connector store](https://github.com/mifrazmurthaja/identity-inbound-auth-jwtsso/releases).
+
+### Configuring the application
+
+1. Enable JWT SSO on your application. Refer to your application's documentation for instruction on enabling JWT SSO
```
Comment (gomathyk) [inline docs/config.md]: instructions

File: docs/config.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    * API key
+    * Logout URL (The URL to be redirected to after the successful logout)
+    * JWS algorithm (Optional, Default: HS256)
+    * JWT parameter name (Optional, Default: jwt)
+    * Redirect URL parameter name (Optional, Default: return_to)
+    * Error URL parameter name (Optional, Default: error_url)
+
+### Deploying JWT SSO artifacts
+
+1. Place the org.wso2.carbon.identity.sso.jwt-x.x.x.jar file into the <IS_HOME>/repository/components/dropins
```
Comment (gomathyk) [inline docs/config.md]: `org.wso2.carbon.identity.sso.jwt-x.x.x.jar`
`<IS_HOME>/repository/components/dropins`

Please give the folder paths and file names in monospace.

File: docs/config.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+ directory.
+2. Add the configuration below in **deployment.toml** file resides in <IS_HOME>/repository/conf directory.
```
Comment (gomathyk) [inline docs/config.md]: the `deployment.toml` file that resides in the `<IS_HOME>/repository/conf` directory.

File: docs/config.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    | --- | --- | --- | --- |
+    | Relying Party | The name of the relying party. This will be needed when we perform authentication request. |  | test-app |
+    | Endpoint API | The endpoint where the JWT response should be sent to after authenticating the user. |  | https://your-app.com/api/sso/v2/sso/jwt |
+    | API Key | The API Key used to sign the JWT token. |  | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+    | JWT Token Expiration Period | The token expiration (exp) to set since the token was generated in seconds. | 120 | 60 |
+    | JWT Signing Algorithm | The JWS Algorithm used to sign the token. | HS256 | Supported Algorithms: HS256/ HS384 / HS512 |
+    | Redirect URL Regex | The regex to validate the Redirect URL. |  | https://your-app.com/.* |
+    | Error URL Regex | The regex to validate the Error URL. |  | https://your-app.com/.* |
+    | Logout URL | The logout URL to be redirected to after the successful logout in WSO2 IS. |  | https://your-app.com/users/sign_out |
+    | JWT Parameter Name | The JWT parameter name to be used to send the JWT token after the authenticating the user. | jwt | jwt |
```
Comment (gomathyk) [inline docs/config.md]: after authenticating the user.

File: docs/config.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    | Redirect URL Parameter Name | The Redirect URL parameter name to be used to include the Redirect URL. | return_to | return_to |
+    | Error URL Parameter Name | The Error URL parameter name to be used to include the Error URL. | error_url | error_url |
+8. Go to **Claim Configuration** and click **Define Custom Claim Dialect** to define the claims to generate the JWT
+ token. (This is required to include the user claims in the JWT token; otherwise, no attributes will be included in
+ the JWT token apart from the default attributes such as exp and iat.) Add the **Service Provider Claim** name that
+ corresponds to the **Local Claim** URI and mark it as **Mandatory Claim**. You'll have to refer to your application's
```
Comment (gomathyk) [inline docs/config.md]: You need to refer

File: docs/config.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+1. Place the org.wso2.carbon.identity.sso.jwt-x.x.x.jar file into the <IS_HOME>/repository/components/dropins
+ directory.
+2. Add the configuration below in **deployment.toml** file resides in <IS_HOME>/repository/conf directory.
+    ````
+    [[resource.access_control]]
+    context="/identity(.*)"
+    secure="false"
+    http_method="GET"
+    ````
+3. [Start/ Restart the WSO2 Identity Server](https://is.docs.wso2.com/en/5.10.0/setup/running-the-product).
```
Comment (gomathyk) [inline docs/config.md]: Can we please make the URLs https://is.docs.wso2.com/en/latest instead of https://is.docs.wso2.com/en/5.10.0?

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: docs/config.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+![JWT SSO Configuration](images/image2.png)
+6. Enter the **Logout URL** to which the user should be redirected after the successful logout.
+7. Refer below for the properties used in the JWT SSO inbound authenticator and the sample values related to those
+ properties.
+    | Field | Description | Default Value | Sample Value |
+    | --- | --- | --- | --- |
+    | Relying Party | The name of the relying party. This will be needed when we perform authentication request. |  | test-app |
+    | Endpoint API | The endpoint where the JWT response should be sent to after authenticating the user. |  | https://your-app.com/api/sso/v2/sso/jwt |
+    | API Key | The API Key used to sign the JWT token. |  | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+    | JWT Token Expiration Period | The token expiration (exp) to set since the token was generated in seconds. | 120 | 60 |
```
Comment (gomathyk) [inline docs/config.md]: The token expiration (exp) to be set. (you can omit the remaining)

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: docs/config.md (hunk 8 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 9 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 7 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 6 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 5 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 4 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 3 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/config.md (hunk 2 above)
Comment (mifrazmurthaja) [reply docs/config.md]: Fixed

File: docs/README.md (hunk 1 above)
Comment (mifrazmurthaja) [reply docs/README.md]: Fixed

**Review by gomathyk — APPROVED** [review]

**Review by Kanapriya — APPROVED** [review]

**Pull request merged**